server/src/main/java/org/opensearch/bootstrap/Security.java

```diff
@@ -402,6 +405,10 @@ static void addFilePermissions(Permissions policy, Environment environment) thro
     private static void addBindPermissions(Permissions policy, Settings settings) {
         addSocketPermissionForHttp(policy, settings);
         addSocketPermissionForTransportProfiles(policy, settings);
+
+        if (FeatureFlags.isEnabled(GRPC_ENABLE_SETTING)) {
```

Comment:
I believe the bootstrap should know nothing about GRPC optional plugin

Reply:
Removed this redundant feature flag.